packages/zoe/src/internal-types.js

```diff
@@ -175,27 +178,44 @@
 
 /**
  * @typedef {object} ZCFRoot
- * @property {ExecuteContract} executeContract
- *
- * @typedef {object} ExecuteContractResult
+ * @property {StartZcf} startZcf
+ * @property {RestartContract} restartContract
+ */
+
+/**
+ * @typedef {object} ExecuteClassicContractResult
  * @property {object} creatorFacet
  * @property {Promise<Invitation>} creatorInvitation
  * @property {object} publicFacet
  * @property {HandleOfferObj} handleOfferObj
  */
 
 /**
- * @callback ExecuteContract
- * @param {SourceBundle} bundle
- * @param {ERef<ZoeService>} zoeService
- * @param {Issuer} invitationIssuer
+ * @typedef {object} ExecuteUpgradeableContractResult
```

Comment:
"classic" and "upgradeable" are squishy to me. "classic" is relative. (see above) "Upgradeable" is not to me well defined, but maybe I'm missing something.

Is the omission of `creatorInvitation` what makes it upgradeable? How?